polar-core/Cargo.toml

```diff
@@ -17,17 +17,14 @@ name = "bench"
 harness = false
 
 [dependencies]
-anyhow = "1.0.33"
-js-sys = "0.3.45"
 lalrpop-util = "0.19.1"
-lazy_static = "1.4.0"
-regex = "1.4.0"
-serde = { version = "1.0.116", features = ["derive", "rc"] }
-serde_json = "1.0.58"
+regex = "1.4.1"
+serde = { version = "1.0.117", features = ["derive", "rc"] }
+serde_json = "1.0.59"
 
 [build_dependencies]
 serde_derive = "1.0"
-lalrpop = { version = "0.19.1", features = ["lexer"] }
```

Comment:
We use our own lexer.

Which also means we no longer need lazy static for the parser, since there are no regexes to compile on init.